### PR TITLE
Altair utils to make graphs brand compliant and to save graphs

### PR DESCRIPTION
**Pull request by india-kerle**
from @beingkk 's have-a-go altair demo

### CODE REVIEW

File: ai_genomics/utils/plotting.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import pandas as pd
+
+ChartType = alt.vegalite.v4.api.Chart
+
+# Fonts and colours
+FONT = "Averta Demo"  # should be changed depending on what your font is called in your system's font book
+TITLE_FONT = "Averta Demo"  # should be changed depending on what your font is called in your system's font book
+FONTSIZE_TITLE = 16
+FONTSIZE_SUBTITLE = 13
+FONTSIZE_NORMAL = 13
```
Comment (Juan-Mateos) [inline ai_genomics/utils/plotting.py]: I would turn these into parameters in the `configure_plots` function so the user has some control over the size of labels and titles in their chart.

File: ai_genomics/utils/plotting.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+Functions for Nesta brand compliant generation of graphs
+"""
+
+import altair as alt
+import pandas as pd
+
+ChartType = alt.vegalite.v4.api.Chart
+
+# Fonts and colours
+FONT = "Averta Demo"  # should be changed depending on what your font is called in your system's font book
```
Comment (Juan-Mateos) [inline ai_genomics/utils/plotting.py]: This didn't work for me. I had to change it to `Averta`

File: ai_genomics/utils/plotting.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            },
+        }
+    }
+
+
+alt.themes.register("nestafont", nestafont)
+alt.themes.enable("nestafont")
+
+
+def configure_plots(fig, chart_title: str = "", chart_subtitle: str = ""):
```
Comment (Juan-Mateos) [inline ai_genomics/utils/plotting.py]: Suggestion: add fontsize parameters with sensible default values e.g. those above

File: ai_genomics/utils/save_plotting.py [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+Utils for easier exporting of altair charts
+"""
+from altair_saver import save
+from selenium import webdriver
+from webdriver_manager.chrome import ChromeDriverManager
+import os
+from typing import Iterator
+from pathlib import Path
+
+FIGURE_PATH = Path("outputs/")
```
Comment (Juan-Mateos) [inline ai_genomics/utils/save_plotting.py]: This saves the files to a new `outputs` folder in the directory where we are running the script. What about...

```
from ai_genomics import PROJECT_DIR

etc.

FIGURE_PATH = Path(f"{PROJECT_DIR}/outputs/figures")
```

**Review by Juan-Mateos — CHANGES_REQUESTED** [review]
Thanks for adding this!

The code broadly runs although I have suggested a couple of changes to give the user a bit more control over the size of labels and titles in their plot, and to ensure we save figures in the right place.

File: ai_genomics/utils/plotting.py (hunk 1 above)
Comment (india-kerle) [reply ai_genomics/utils/plotting.py]: good idea

**Comment by india-kerle** [comment]
@Juan-Mateos thanks for your comments - I've made your suggested changes. Let me know if its ok to merge as a result.

**Comment by Juan-Mateos** [comment]
Looks good, thank you! Merged!

**Pull request merged**